lucida/botframework-interface/credentials.template

```diff
@@ -0,0 +1,6 @@
+// Copy application ID and password (as explained in README.md) and rename this file to credentials.js
```

Comment:
I prefer it this way because then we can put credentials.js in .gitignore and not worry abt passwords being pushed